spotbugs-exclude.xml

```diff
@@ -6,4 +6,9 @@
       <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE"/>
     </Or>
   </Match>
+  <Match>
```

Comment:
You should revert changes to this file and instead remove https://github.com/jenkinsci/ec2-plugin/blob/1552e0d4d3941215e0258fa352c003413cd69f28/src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java#L116

Reply:
Sure, will give that a shot and see if Spotbugs is happy with that